packages/editor/src/components/modals/projectTemplateModal/templates/index.ts

```diff
@@ -36,7 +36,7 @@ export default class Templates {
         {
             id: 2,
             name: 'CryptoPizza NFT',
-            description: '<div><h2>CryptoPizza NFT</h2><p>A Crypto-collectible game built on top of the ERC-721 standard for creating unique tokens.</p></div>',
+            description: '<div><h2>CryptoPizza NFT</h2><p>A game built on top of the ERC721 standard for creating unique collectible tokens.</p></div>',
```

Comment:
Droped the dash from `ERC721`. Not sure if there is an established convention around this? I'm just following OpenZeppelin on this:
https://docs.openzeppelin.com/contracts/2.x/api/token/erc721